book/running-wallaroo/resilience-data-redundancy.md

```diff
@@ -0,0 +1,218 @@
+# Data Redundancy for Wallaroo's Crash Recovery
```

Comment:
This file seems to have been word-wrapped at 80 columns. Typically don't do that.

Reply:
I'll unwrap.